### PR TITLE
Do not quote arguments to ST_EstimatedExtent

### LINKED ISSUE #252: Arguments to ST_EstimateExtent must not be quoted
`_cdb_estimated_extent` generates a call to `ST_EstimateExtent` quoting (with double quotes) its arguments when they need it as identifiers, but that causes a syntax error when `ST_EstimateExtent` evaluates the arguments

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #252

### CODE REVIEW

**Comment by jgoizueta** [comment]
@rafatower  @ethervoid  please take a look

File: scripts-available/CDB_Overviews.sql [hunk 1]
```diff
@@ -235,7 +235,7 @@ AS $$
       FROM pg_class c JOIN pg_namespace n on n.oid = c.relnamespace WHERE c.oid = reloid::oid;
 
     ext_query = format(
-      'SELECT ST_EstimatedExtent(''%1$I'', ''%2$I'', ''%3$I'');',
+      'SELECT ST_EstimatedExtent(''%1$s'', ''%2$s'', ''%3$s'');',
```
Comment (rafatower) [inline scripts-available/CDB_Overviews.sql]: Wondering why a dynamic query is needed here. Can't you just use `RETURN QUERY`?

Comment (rafatower) [inline scripts-available/CDB_Overviews.sql]: never mind, if it works go ahead

**Comment by rafatower** [comment]
:+1:

**Comment by ethervoid** [comment]
Seems good to me

**Pull request merged**